Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSecurityEvents.yaml

```diff
@@ -13,76 +13,134 @@ References:
-                  TargetProcessId = tostring(toint(NewProcessId)),
-                  TargetProcessName = NewProcessName,
-                  TargetProcessCommandLine = CommandLine,
-                  TargetProcessTokenElevation = TokenElevationType
-                | lookup MandatoryLabelLookup on MandatoryLabel 
-              // -- Aliases
-                | extend
-                  User = TargetUsername,
-                  Dvc = DvcHostname,
-                  Process = TargetProcessName
-              // -- Remove potentially confusing
-                | project-away 
-                    ParentProcessName, 
-                    ProcessId, 
-                    SubjectUserSid,
-                    SubjectDomainName,
-                    SubjectUserName,
-                    SubjectAccount,
-                    TargetUserSid,
-                    TargetDomainName,
-                    TargetUserName,
-                    TargetAccount
-              }; ProcessEvents
+    let MandatoryLabelLookup = datatable (MandatoryLabel:string,MandatoryLabelRid:string, MandatoryLabelText:string, MandatoryLabelMeaning:string)
+      [
+        'S-1-16-0', '0x00000000', 'SECURITY_MANDATORY_UNTRUSTED_RID', 'Untrusted',
+        'S-1-16-4096', '0x00001000', 'SECURITY_MANDATORY_LOW_RID', 'Low integrity',
+        'S-1-16-8192', '0x00002000', 'SECURITY_MANDATORY_MEDIUM_RID', 'Medium integrity',
+        'S-1-16-8448', '0x00002100', 'SECURITY_MANDATORY_MEDIUM_PLUS_RID', 'Medium high integrity',
+        'S-1-16-12288', '0X00003000', 'SECURITY_MANDATORY_HIGH_RID', 'High integrity',
+        'S-1-16-16384', '0x00004000', 'SECURITY_MANDATORY_SYSTEM_RID', 'System integrity',
+        'S-1-16-20480', '0x00005000', 'SECURITY_MANDATORY_PROTECTED_PROCESS_RID', 'Protected process'
+      ];
+    let ProcessEvents=(){
+    SecurityEvent
+    // --------------------------------------------------------------------------------------
+    | where
+      (isnull(starttime) or TimeGenerated >= starttime )
+      and (isnull(endtime) or TimeGenerated >= endtime )
+      and not(disabled)
+      and (array_length(commandline_has_all)==0 or CommandLine has_all (commandline_has_all))
+      and (array_length(commandline_has_any)==0 or CommandLine has_any (commandline_has_any))
+      and (array_length(commandline_has_any_ip_prefix)==0 or has_any_ipv4_prefix(CommandLine, commandline_has_any_ip_prefix) )
+      and (array_length(actingprocess_has_any)==0 or ParentProcessName has_any (actingprocess_has_any))
+      and (array_length(targetprocess_has_any)==0 or NewProcessName has_any (targetprocess_has_any))
+      and (array_length(parentprocess_has_any)==0)  /// ????
+      and (targetusername=='*' or TargetAccount has targetusername)
+      and (array_length(dvcipaddr_has_any_prefix)==0)
+      and (array_length(dvcname_has_any)==0 or Computer has_any (dvcname_has_any))
+      and (eventtype=='*' or eventtype=='ProcessCreated')
```

Comment:
Should be first. Missed it for the previous PR (LinuxSysmon)

Reply:
true